docs/source/key_components/rl_toolkit.rst

```diff
@@ -176,6 +195,20 @@ To performing a single gradient step on the model, call the ``step`` function:
 Here it is assumed that the losses have been computed using the same model instance and the gradients have
 been generated for the internal components.  
 
+Additionally, core model provides ``get_gradients`` and ``apply_gradients`` interfaces for finer control in
+model update, for example, distributed training. Here is an example code of gradient related interfaces:
+
+.. code-block:: python
+
+    grad_dict_list = []
+    # gathering gradient from multiple sources
+    grad_dict_list.append(ac_model_0.get_gradients(loss_0))
+    grad_dict_list.append(ac_model_1.get_gradients(loss_1))
+    ac_model.apply_gradients(grad_dict_list)
+
+The ``get_gradients`` function returns a gradient dict, and the ``apply_gradients`` function takes a list of
```

Comment:
The averaging is done at the policy level now. ``apply_gradients`` takes a single grad_dict instead of a list.